### DISCUSSION TITLE
ip of users

### DISCUSSION THREAD

**Discussion by 1gzify** [Paymenter/Paymenter#208] [Ideas]
make it so we can see the ip of users to see if they are making alt acc

**Comment by finnjosten** [comment]
i mean a vpn would bypass this

**Reply by 1gzify** [reply]
could make a anti vpn

**Reply by finnjosten** [reply]
And then let basicly not even half of the internet in

**Comment by BasToTheMax** [comment]
I think an anti-alt in general would be better

**Comment by cloudrack-ca** [comment]
[fraudlabspro...? ](https://www.fraudlabspro.com/) 0w0 its free
